docs/get-started/3d-mesh/index.md

```diff
@@ -0,0 +1,303 @@
+
+In this tutorial we will finetune PointConv on [ModelNet40 dataset](https://arxiv.org/pdf/1406.5670v3.pdf).
+The ModelNet40 dataset contains synthetic object point clouds. As the most widely used benchmark for point cloud analysis, ModelNet40 is popular because of its various categories, clean shapes, well-constructed dataset.
+
+The original ModelNet40 consists of 12,311 CAD-generated meshes in 40 categories (such as airplane, car, plant, lamp), of which 9,843 are used for training while the rest 2,468 are reserved for testing.
+
+```{tip}
+ModelNet: a consistent, large-scale dataset of 3D objects annotated with fine-grained, instance-level, and hierarchical 3D part information.
+The corresponding point cloud data points are uniformly sampled from the mesh surfaces, and then further preprocessed by moving to the origin and scaling into a unit sphere. This dataset is used to establish three benchmarking tasks for evaluating 3D part recognition: fine-grained semantic segmentation and classification. In our case we'll use it to finetune PointConv which was pretrained on [ShapeNet dataset](https://arxiv.org/abs/1512.03012).
+```
+
+You can download ModelNet dataset on the official [Princeton website](https://modelnet.cs.princeton.edu/).
+
+```shell
+wget http://modelnet.cs.princeton.edu/ModelNet40.zip
+
+```
+
+## Data preparation
+
+After downloading ModelNet data we need to unzip it.
+
+```shell
+pip install finetuner
+pip install trimesh
+
+unzip ModelNet40.zip
+```
+
+Before we go further let's look at some examples from our data:
+
+```{figure} airplane.gif
+:align: center
+:width: 70%
+```
+
+```{figure} lap.gif
+:align: center
+:width: 70%
+```
+
+Now let's go to loading our data. We'll use `DocumentArray` from the library `docarray`. We'll go through all the files in our directory.
+And we'll recursively read all files that end with `.off`. We split the files between two `DocumentArray`s depending on their uri to train
+and test.
+
+Finally we save the resulting `DocumentArray`s into binary files for later use.
+
+```python
+import glob
+import os
+from typing import Optional
+
+import trimesh
+from docarray import Document, DocumentArray
+
+train = DocumentArray()
+test = DocumentArray()
+data_path = 'ModelNet40'
+all_docs = DocumentArray.from_files(os.path.join(data_path,'/**/**/*.off'))
+for doc in all_docs:
+    if 'test' in doc.uri :
+        test.append(doc)
+    else:
+        train.append(doc)
+
+train.save_binary('train_data_modelnet.bin')
+test.save_binary('test_data_modelnet.bin')
+
+```
+
+## Model Training
+
+Now that we have our data ready, we need a model that creates embeddings we will later use those embeddings for searching similar matches for different queries. A lot of work and research has been done in the field of 3D data embeddings there are some powerful models like [PointConv](https://arxiv.org/abs/1811.07246) which have been trained on [ShapeNet dataset](https://arxiv.org/pdf/1512.03012.pdf)
```

Comment:
A lot of work and research has been done in the field of 3D data embeddings **and** there are some powerful models